docs/sections/user_guide/yaml/components/cdeps.rst

```diff
@@ -0,0 +1,53 @@
+.. _cdeps_yaml:
+
+cdeps
+=====
+
+Structured YAML to run :CDEPS:`cdeps<>` is validated by JSON Schema and requires the ``cdeps:`` block, described below.
+
+.. include:: /shared/injected_cycle.rst
+
+Here is a prototype UW YAML ``cdeps:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: /shared/cdeps.yaml
+
+UW YAML for the ``cdeps:`` Block
+--------------------------------
+
+atm_in
+^^^^^^
+
+Configures the atm namelist file. Supports ``base_file:`` and ``update_values:`` blocks (see :ref:`updating_values` for details). Namelist options are described :cdeps:`??? HERE ???<>`.
+
+.. include:: /shared/validate_namelist.rst
+
+atm_streams
+^^^^^^^^^^^
+
+Configures the atm streams.
+
+  **streams:**
+
+  The ``streams:`` block may contain sub-blocks ``stream01`` through ``stream09``. Each such sub-block specifies a stream configuration to be used to render the Jinja2 template file named by ``template_file:`` (see below). Supported keys and avlues are described :cdeps:`??? HERE ???<>`.
```

Comment:
Here, too, we should add descriptions of the stream-related UW YAML items. Since these are currently variables for use in a Jinja2 template, maybe there's no appropriate external documentation, in which case we'll just need to list and describe them here (but possibly back in the main PR targeting the `ufs-community` `uwtools` repo).  But if there's external documentation we can reference, we could possibly get away with just a table mapping our UW YAML names (`dtlimit`, `mapalgo`, etc.) to the official docs. Or, if there's a one-to-one mapping from UW YAML names to documented names, that's even better.

Essentially, though, we need to document what variables need to be set in YAML in e.g. `cdeps.atm_streams.streams0`.